pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -350,8 +355,8 @@ func (i *CommandInitOption) createCertsSecrets() error {
 }
 
 func (i *CommandInitOption) initKarmadaAPIServer() error {
-	// wait karmada APIServer component ready timeout 120s
-	waitKarmadaAPIServerComponentReadyTimeout := 120
+	// wait karmada APIServer component ready timeout i.WaitComponentReadyTimeout*4
+	waitKarmadaAPIServerComponentReadyTimeout := i.WaitComponentReadyTimeout * 4
```

Comment:
Any reason for `WaitComponentReadyTimeout * 4`  and not `WaitComponentReadyTimeout`?

Reply:
Because it was: originally, waitKarmadaAPIServerComponentReadyTimeout=120s (cmdinitoptions. WaitComponentReadyTimeout (30s) * 4), I guess this may have been set due to the slow startup of KarmadaAPIServer.
So I don't want to change this logic.